birdyboard.py

```diff
@@ -0,0 +1,421 @@
+import pickle
+from chirp import *
+from thread import *
+from user import *
+
+
+class Birdyboard:
+    """
+    this app's functionality is a private/public message board. It constructs three subclasses which pull from three serialized .txt files: users, threads, and chirps. All items in these three files are identified by a unique UUID.
```

Comment:
Technically the .txt files aren't serialized, the data is.  It'd be more correct to say "pull serialized data from three .txt files"